ngraph/frontend/onnx_import/src/op/upsample.cpp

```diff
@@ -70,6 +75,73 @@ namespace ngraph
             }
 
             namespace set_1
+            {
+                OutputVector upsample(const onnx_import::Node& node)
+                {
+                    const auto height_scale = node.get_attribute_value<float>("height_scale");
+                    const auto width_scale = node.get_attribute_value<float>("width_scale");
+                    const auto mode = node.get_attribute_value<std::string>("mode", "nearest");
+                    check_mode_support(node, mode, 1);
+
+                    const auto interpolate_mode = convert_mode(mode);
+                    const auto zero_pad = std::vector<size_t>(1, 0);
```

Comment:
```suggestion
                    const std::vector<size_t> zero_pad = {0};
```

Reply:
I like this change more than mine :)
but '=' is unnecessary, isn't it?
```suggestion
                    const std::vector<size_t> zero_pad{0};
```